Server/Models/Item.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Karma.Server.Models
+{
+    public class Item
+    {
+        public string Name { get; set; }
+
+        public string Description { get; set; }
+
+        public ItemState State { get; set; }
+
+        public int Id { get; set; }
+
+        public Item ()
+        {
+
+        }
+
```

Comment:
Why is there an empty constructor that is empty?

Reply:
When I tested file reading I got an error that said something about having a constructor that does not have any parameters(don't remember the exact phrasing). The error was resolved by writing this empty constructor. It looked like a decent fix for me, but maybe there is a better way to fix this?

Reply:
Maybe it's because you tried to create object by using empty constructor, which does not exist if parameterized constructor is created?

Reply:
Let me check really quickly

Reply:
It says "NotSupportedException: Deserialization of reference types without parameterless constructor is not supported. Type 'Karma.Server.Models.Item'"

Reply:
Fixed